### PR TITLE
Add toString for DirectCredentials

### PR DESCRIPTION
**Pull request by raboof**
To make 'show credentials' more useful

### CODE REVIEW

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?sbt=develop&zinc=develop&io=develop&librarymanagement=pull/303/head&util=develop&website=develop&deploy_to_bintray=false&run_sbt_community=false&verbose_dbuild=false&community_verbose=false">this link.</A>

File: ivy/src/main/scala/sbt/librarymanagement/ivy/Credentials.scala [hunk 1]
```diff
@@ -83,4 +83,5 @@ final class DirectCredentials(
     val host: String,
     val userName: String,
     val passwd: String
+  override def toString = s"DirectCredentials('$realm', '$host', '$userName', ****)"
```
Comment (eed3si9n) [inline ivy/src/main/scala/sbt/librarymanagement/ivy/Credentials.scala]: `val passwd: String` is in the param list, so you need `) extends Credentials { ... }` block.

**Review by eed3si9n — CHANGES_REQUESTED** [review]

File: ivy/src/main/scala/sbt/librarymanagement/ivy/Credentials.scala (hunk 1 above)
Comment (raboof) [reply ivy/src/main/scala/sbt/librarymanagement/ivy/Credentials.scala]: Doh sorry :/

**Review by eed3si9n — APPROVED** [review]
Thanks!

**Pull request merged**